src/traces/scatter/attributes.js

```diff
@@ -67,7 +67,11 @@ module.exports = {
     },
     ids: {
         valType: 'data_array',
-        description: 'A list of keys for object constancy of data points during animation'
+        description: [
```

Comment:
Actually, I think it's best to leave unspecified… It could be country and you could filter by inclusion in a list, or it could be a value and you could filter numerically.